pages/k8s/overview.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Overview | Canonical Distribution of Kubernetes&reg;"
+keywords: juju, kubernetes, cdk
+tags: [intro]
+sidebar: k8smain-sidebar
+permalink: overview.html
+layout: base
+toc: False
+---
+
+# Overview
+
+<div class="p-content__row">
+    <div class="u-equal-height">
+        <div class="col-6">
+
+<p>Deliver ‘Containers as a Service’ across the enterprise with the
+<strong>Canonical Distribution of Kubernetes<sup>&reg;</sup> </strong>
+(<strong>CDK</strong>) , enabling each project to
+spin up a standardised K8s of arbitrary scale, on demand, with centralised
+operational control. <strong>CDK</strong>  provides a
+well integrated, turn-key <strong>Kubernetes<sup>&reg;</sup></strong>
+platform that is open, extensible, and secure.
+</p>
+<p>
+For more information on the features and benefits of **CDK**, including details
```

Comment:
This doesn't render bold for some reason.

Reply:
yes, because it is in an html block. will fix that, thanks